server/backend/cron/feedback_email.go

```diff
@@ -0,0 +1,139 @@
+package cron
+
+import (
+	"bytes"
+	"crypto/tls"
+	htmlTemplate "html/template"
+	"os"
+	"strconv"
+	textTemplate "text/template"
+	"time"
+
+	"github.com/TUM-Dev/Campus-Backend/server/model"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/gomail.v2"
```

Comment:
nit: Can we stick to normal github URLs please instead of using this weird indirection?

Reply:
Oh, it doesn't even support go modules :(

Reply:
I could not find a way to do so

Reply:
Yeah, nevermind. Probably the best way forward would be to not use it at all, but ugh, whatever.